.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,14 @@
-**Replace this placeholder text with a summary of the changes in your PR.
-The more detailed you are, the better.**
+**Replace this placeholder text with a summary of the changes in your PR.**
 
 -----------------
 
-Before submitting a PR make sure the following things have been done (and denote this
-by checking the relevant checkboxes):
+Before submitting a PR mark the checkboxes for the items you've done (or if you
+think they don't apply):
 
-- [ ] The commits are consistent with our [contribution guidelines][1]
-- [ ] You've added tests (if possible) to cover your change(s). Indentation & font-lock tests are extremely important!
-- [ ] All tests are passing (`make test`)
-- [ ] The new code is not generating bytecode or `M-x checkdoc` warnings
-- [ ] You've updated the changelog (if adding/changing user-visible functionality)
-- [ ] You've updated the readme (if adding/changing user-visible functionality)
+- [ ] The commits are consistent with our [contribution guidelines][1].
+- [ ] Optionally add a test or two. This is specially important when fixing bugs, or when changing indentation or font-locking.
```

Comment:
I don't see how this reworded version is better than the original.

Reply:
It was just to add the word "optionally".